packages/symfony-static-dumper/src/Application/SymfonyStaticDumperApplication.php

```diff
@@ -35,13 +38,19 @@ public function __construct(
         $this->assetsCopier = $assetsCopier;
     }
 
-    public function run(string $publicDirectory, string $outputDirectory): void
```

Comment:
I'm not sure how strict BC Breaks policy symplify has but if that's a problem I can restore that method, add `routeNames` into it and use `dumpControllers` and `copyAssets` inside of it.